lib/fog/ovirt/requests/compute/v4/create_vm.rb

```diff
@@ -40,14 +40,21 @@ def create_vm(attrs)
 
             attrs[:comment] ||= ""
             attrs[:quota] = attrs[:quota].present? ? client.system_service.data_centers_service.data_center_service(datacenter).quotas_service.quota_service(attrs[:quota]).get : nil
+            cpu_topology = OvirtSDK4::CpuTopology.new(:cores => attrs[:cores], :sockets => attrs[:sockets])
+
+            if attrs[:cores].present? || attrs[:sockets].present?
+              attrs[:cpu] = OvirtSDK4::Cpu.new(:topology => cpu_topology)
+            end
+
+            cpu_topology = OvirtSDK4::CpuTopology.new(:cores => attrs[:cores], :sockets => "1") if attrs[:cores].present?
```

Comment:
Here you are ignoring the sockets. Is that intentional? If so, I'd create one if/else block and assign `attrs[:cpu]` just once. The rubocop is IMHO right and it's hard to exactly follow what's going.